network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -1047,7 +1046,7 @@ def exec(self):
         explanation_not_contained_other_self = \
             contained_2_in_1.output_result + ':\n\t' + contained_2_in_1.output_explanation
         if contained_1_in_2.bool_result and contained_2_in_1.bool_result:
-            return QueryAnswer(bool_result=False,
+            return QueryAnswer(bool_result=True,
```

Comment:
Add a relevant test that will pass following this update and fail before this update.

Reply:
- when having a test from a scheme file the important vlaue to be compared with the expected is the numerical result. so the bool result doesn't affect the final returned value in this case (meaningless)
- for cmdline tests the result is the "not bool" which means when there is a two way containment (True) we expect a "0" returned value (success) --> added the "two_way_containment_policies" test to the k8s_cmdline_tests.yaml file

the output of running the new query on both branches (this and master) will be :

>The two network configurations C:/Users/ShireenFalah/network-config-analyzer/tests/k8s_testcases/example_policies/testcase1/testcase1-networkpolicy1.yaml and C:/Users/ShireenFalah/network-config-analyzer/tests/k8s_testcases/example_policies/testcase1/testcase1-networkpolicy2.yaml are semantically equivalent.

The difference: 
on this branch it exit with Process finished with exit code 0
while on master branch Process finished with exit code 1